### PR TITLE
Don't override Name tags of EBS volumes

### PR DESCRIPTION
**Pull request by a1exsh**
Sometime we rely on Name tag of the data EBS volume, which is different from
the usual 'spilo_{cluster_name}' format, for example, when attaching multiple
volumes and making a RAID on top of them.

This will ensure that any pre-defined Name tags are kept intact.  The instance
and role tags are updated in any case.

### CODE REVIEW

**Comment by alexeyklyukin** [comment]
Does it make sense to pick another tag for the spilo cluster name?

File: postgres-appliance/callback_aws.py [hunk 1]
```diff
@@ -67,11 +61,15 @@ def main():
...
+    tag_resource(ec2, instance_id, tags)
+
+    tags.update({'Instance': instance_id})
+    name_tag = 'spilo_' + cluster
 
-    tags.update({'Instance': instance_id, 'Name': 'spilo_' + cluster})
     volumes = list_volumes(ec2, instance_id)
-    tag_volumes(ec2, volumes, tags)
+    for v in volumes:
+        tags_to_update = tags if 'Name' in v.tags else dict(tags, Name=name_tag)
```
Comment (CyberDem0n) [inline postgres-appliance/callback_aws.py]: Should we differentiate between root volume and data volumes?

Comment (a1exsh) [inline postgres-appliance/callback_aws.py]: Well, we didn't do that before.  If we start now, that will introduce inconsistencies.

**Comment by a1exsh** [comment]
@alexeyklyukin currently the Name tag is only used for sort/filtering in the AWS console.  If we would use a different tag name, the console will need to be customized by every user to display it.

**Comment by alexeyklyukin** [comment]
@a1exsh but if we don't - wouldn't it lead to inconsistent results when filtering/sorting on that tag?

**Comment by avaczi** [comment]
From a billing perspective, the spilo cluster name (and a tag for the team, but that might be too complicated to solve on this level) would be a very good idea @alexeyklyukin

**Comment by CyberDem0n** [comment]
:+1:

**Comment by a1exsh** [comment]
:+1:

**Pull request merged**